Tests/IGListSectionMapTests.m

```diff
@@ -23,39 +23,39 @@ @implementation IGListSectionMapTests
 
 - (void)test_whenUpdatingItems_thatArraysAreEqual {
     NSArray *objects = @[@0, @1, @2];
-    NSArray *sectionControllers = @[@"a", @"b", @"c"];
+    NSArray *sectionControllers = @[[IGListTestSection new], [IGListTestSection new], [IGListTestSection new]];
```

Comment:
all of these are just so that the test actually respects the types that `IGListSectionMap` require.